ADMIN.adoc

```diff
@@ -0,0 +1,35 @@
+= Repository Administrator Responsibilities
+
+== Administrator rotation schedule:
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be administrator is voluntary.
+
+== Administrator responsibilities:
+
+As the administrator, you are in charge of performing the following duties:
+
+=== Being the Point of Contact for general issues related to the Supplementary Style Guide repository
+
+* You are the point of contact in case other contributors experience issues with the repository.
+When people reach out to you about an issue, try to fix it. If you cannot fix the issue, report it to a member of the initiative who can.
+
+=== Maintaining the Style Guide site on GitHub
+
+* Ensure that the link:https://redhat-documentation.github.io/supplementary-style-guide/[style guide site] is available and displayed correctly.
+* Create pull requests to update the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/README.md[README.md], link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/GUIDELINES.adoc[GUIDELINES.adoc] and the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/CONTRIBUTING.md[Contribution guide], if necessary.
+
+=== Driving resolution of link:https://github.com/redhat-documentation/supplementary-style-guide/issues[open issues]
```

Comment:
I don't think there should be links in headings. You already have this link in the first bullet.

In general, I think that having duplicate links in the same topic is not user-friendly.

Reply:
+1

Reply:
@stoobie @JoanneOFlynn2018 Got it, I moved the links form headings to regular text and I removed the duplicates.